docusaurus/website/sidebars.js

```diff
@@ -13,55 +13,62 @@
 
 /** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
 const sidebars = {
-  // By default, Docusaurus generates a sidebar from the docs folder structure
-  // tutorialSidebar: [{ type: 'autogenerated', dirName: '.' }],
-
-  docs: {
-    Introduction: [
-      'introduction/backend-plugins',
-      'introduction/grafana-plugin-sdk-for-go',
-      'introduction/plugin-protocol',
-      'introduction/data-frames',
-    ],
-    'Getting Started': [
-      'getting-started',
-      'creating-a-plugin',
-      'folder-structure',
-      'updating-to-new-releases',
-      {
-        type: 'doc',
-        id: 'migrating-from-toolkit',
-        label: 'Migrating from Toolkit',
-      },
-    ],
-    Development: [
-      'frontend',
-      'backend',
-      {
-        type: 'doc',
-        id: 'docker',
-        label: 'Development Environment',
-      },
-      'ci',
-    ], // TODO: editor-setup, docker
-    // 'Building your Plugin': ['production-builds', ],
-    // Testing: ['unit-tests', 'e2e-tests'],
-    Distribution: ['distributing-your-plugin', 'signing-your-plugin'],
-    'Advanced Usage': ['advanced-configuration', 'nested-plugins'],
-  },
-
-  // But you can create a sidebar manually
-  /*
-  tutorialSidebar: [
-    'intro',
-    'hello',
+  docs: [
+    {
+      type: 'category',
+      label: 'Introduction',
+      collapsed: true,
+      items: [
+        'introduction/backend-plugins',
+        'introduction/grafana-plugin-sdk-for-go',
+        'introduction/plugin-protocol',
+        'introduction/data-frames',
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Getting Started',
+      collapsed: false,
+      items: [
+        'getting-started',
+        'creating-a-plugin',
+        'folder-structure',
+        'updating-to-new-releases',
+        'migrating-from-toolkit',
+      ],
+    },
+    {
+      type: 'category',
+      label: 'Development',
+      collapsed: true,
+      items: [
+        'frontend',
+        'backend',
+        {
+          type: 'doc',
```

Comment:
This section is unnecessary but shows how we could override the default title metadata of a page to change its appearance in the ToC, without changing the target document. Another alternative would be `sidebar_label: string` in the target document.